### PR TITLE
updated sponsor image

### LINKED ISSUE #623: update Ab Initio sponsor logo
They gave us a JPEG 

logo

### PR DESCRIPTION
**Pull request by Wassaf-Shahzad**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #623 

#### What's this PR do?
Updated Ab Initio Logo

#### How should this be manually tested?
Verify sponsor `Ab initio` logo has been updated in home page

#### Screenshots (if appropriate)
Screenshot 2022-04-11 at 10 04 51 AM
Screenshot 2022-04-11 at 10 03 24 AM

### CODE REVIEW

**Review by MAbdurrehman12 — APPROVED** [review]
👍

**Comment by MAbdurrehman12** [comment]
@pdpinch Just a few thoughts: 
1. Is there any need to make these logos clickable and open the sponsor's website in a new tab?
2. I downloaded this MathWorks SVG logo from their website, is there a need to update this too?
mathworks

**Comment by pdpinch** [comment]
> 1. Is there any need to make these logos clickable and open the sponsor's website in a new tab?

No, but thanks for asking. 

> 2. I downloaded this MathWorks SVG logo from their website, is there a need to update this too?

Let's try it!

**Comment by Wassaf-Shahzad** [comment]
@pdpinch  updated mathwork logo need you go ahead for merge

File: www/layouts/home.html [hunk 1]
```diff
@@ -151,10 +151,10 @@ <h2>Your Donation Makes a Difference</h2>
       <h2 class="text-uppercase text-center">Our Corporate and Foundation Supporters</h2>
       <div class="row w-100 justify-content-center">
         <div class="col-12 col-md-3 col-xl-2 d-flex align-items-center justify-content-center py-3">
```
Comment (MAbdurrehman12) [inline www/layouts/home.html:153]:
```suggestion
        <div class="col-12 col-md-3 col-xl-2 d-flex justify-content-center py-md-2 py-3">
```
Improves this alignement:
Screenshot 2022-04-13 at 2 21 35 PM
(note: This change is only for `ab_initio_logo` logo div)

**Comment by MAbdurrehman12** [comment]
Sorry, @Wassaf-Shahzad :/ I should have noticed this earlier but it looks like the first 3 logos are looking good but `andrew-w-mellon-foundation` logo seems to be blurred. 
Hence I tried to find their logo online and found this `jpeg` file and converted it into a `svg`. 

Mellon-Foundation-logo-2022
andrew-w-mellon-foundation

Replacing the current logo with SVG looks good I think:
image

What do you think?

**Comment by pdpinch** [comment]
Wow, that SVG for Mellon came out really well. I agree, let's swap it out and merge this

**Review by MAbdurrehman12 — APPROVED** [review]
👍

**Pull request merged**